core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3756,6 +3757,7 @@ private void startSyncAndJournal(RpcContext rpcContext, AlluxioURI uri)
       try {
         mSyncManager.startSyncPostJournal(uri);
       } catch (Throwable e) {
+        LOG.warn("Start sync failed because {}", ExceptionUtils.getStackTrace(e));
```

Comment:
I think if you just add the exception as the last parameter, the logging library will log the exception and stack trace. Also, we should include the uri.

like `LOG.warn("Start sync failed on {}", uri, e); `